src/libtrx/game/level/common.c

```diff
@@ -214,6 +214,41 @@ void Level_ReadObjectMeshes(
     Vector_Free(unique_indices);
 }
 
+void Level_ReadAnims(
+    const int32_t base_idx, const int32_t num_anims, VFILE *const file,
+    int32_t **frame_pointers)
+{
+    for (int32_t i = 0; i < num_anims; i++) {
+        ANIM *const anim = Anim_GetAnim(base_idx + i);
+#if TR_VERSION == 1
+        anim->frame_ofs = VFile_ReadU32(file);
+        const int16_t interpolation = VFile_ReadS16(file);
+        ASSERT(interpolation <= 0xFF);
+        anim->interpolation = interpolation & 0xFF;
+        anim->frame_size = 0;
+#else
```

Comment:
Are these `else` consistent with all later TRs? Is it worth explicitly saying `#if TR_VERSION == 2` to set up for potentially a `TR3X`?

Reply:
TR3 uses the same format as TR2. Arguably, TR1 is the same in terms of struct size, it's just that frame size was introduced in TR2 to deal with the way rotations are packed.
TR4 and 5 have extra (unrelated) properties (lateral speed and acceleration IIRC).

Reply:
It's OK – we'll have to go through every single `if TR_VERSION` when thinking to support more games anyway.